runtime-version/.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": "google",
+  "installedESLint": true,
+  // http://eslint.org/docs/rules/
```

Comment:
nit: Comments aren't allowed in JSON. A common solution is to use a "_comment" field to add the comment and not end up with an invalid JSON. ESLint doesn't seem to care about this though.